cli/args.go.orig

```diff
@@ -0,0 +1,150 @@
+package cli
```

Comment:
I think you may have accidentally checked in some files `.orig` files, probably from a merge conflict. Could you remove these?